index.js

```diff
@@ -31,11 +31,18 @@ const urlPrefix = "https://github.com";
  */
 
 const toUrlFormat = (item) => {
-  if (typeof item === "object") {
-    return Object.hasOwnProperty.call(item.payload, "issue")
-      ? `[#${item.payload.issue.number}](${urlPrefix}/${item.repo.name}/issues/${item.payload.issue.number})`
-      : `[#${item.payload.pull_request.number}](${urlPrefix}/${item.repo.name}/pull/${item.payload.pull_request.number})`;
+  if (typeof item !== "object") {
+    return `[${item}](${urlPrefix}/${item})`;
   }
+
+  if (Object.hasOwnProperty.call(item.payload, "comment")) {
+    return `[#${item.payload.issue.number}](${item.payload.comment.html_url})`;
+  } else if (Object.hasOwnProperty.call(item.payload, "issue")) {
+    return `[#${item.payload.issue.number}](${item.payload.issue.html_url})`;
+  } else if (Object.hasOwnProperty.call(item.payload, "pull_request")) {
+    return `[#${item.payload.pull_request.number}](${item.payload.pull_request.html_url})`;
+  }
+
   return `[${item}](${urlPrefix}/${item})`;
```

Comment:
```suggestion
  if (Object.hasOwnProperty.call(item.payload, "comment")) {
    return `[#${item.payload.issue.number}](${item.payload.comment.html_url})`;
  }
  if (Object.hasOwnProperty.call(item.payload, "issue")) {
    return `[#${item.payload.issue.number}](${item.payload.issue.html_url})`;
  }
  if (Object.hasOwnProperty.call(item.payload, "pull_request")) {
    return `[#${item.payload.pull_request.number}](${item.payload.pull_request.html_url})`;
  }
```

`Else` is redundant due to the early return.

Reply:
Also, let's move `urlPrefix` inside the `toUrlFormat` method since it's not consumed elsewhere.